### PR TITLE
Report skipped hermione tests

### LINKED ISSUE #653: Skipped hermione tests marked as Unknown in allure report
**Describe the bug**
Skipped hermione tests marked as Unknown in allure report

**To Reproduce**
1. Mark test by skip parameter "hermione.skip.in" or "it.skip"
2. Create allure report

**Expected behavior**
Test marked as skipped

**Screenshots**
image

### PR DESCRIPTION
**Pull request by epszaw**
### Context

fixes #653 

#### Checklist
- [x] [Sign Allure CLA][cla]
- [x] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

File: packages/allure-hermione/src/index.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ export type TestIDFactory = (testId?: string) => string;
 const hostname = os.hostname();
 
 const hermioneAllureReporter = (hermione: Hermione, opts: AllureReportOptions) => {
+  const loadedTests = new Map<string, Hermione.Test>();
```
[the remark this answers is not in the document]
Comment (epszaw) [reply packages/allure-hermione/src/index.ts:45]: Make id's uniq, but keep Map

File: packages/allure-hermione/src/index.ts [hunk 2]
```diff
@@ -227,40 +269,20 @@ const hermioneAllureReporter = (hermione: Hermione, opts: AllureReportOptions) =
       }
     });
   });
+  hermione.on(hermione.events.AFTER_TESTS_READ, (collection) => {
+    // cache all the tests to handle skipped tests in future
+    collection.eachTest((test) => {
+      loadedTests.set(test.fullTitle(), test);
```
[the remark this answers is not in the document]
Comment (epszaw) [reply packages/allure-hermione/src/index.ts]: Done

File: packages/allure-hermione/test/fixtures/skipped.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+it.skip("should be skipped", () => {});
+
+describe("with skip", () => {
+  it.skip("should be skipped", () => {});
+});
+
```
[the remark this answers is not in the document]
Comment (epszaw) [reply packages/allure-hermione/test/fixtures/skipped.js:6]: They actually have problem with tests only, let's keep this implementation only for tests and add suites handling later

**Review by vovsemenv — APPROVED** [review]
Lestgooo

**Pull request merged**